### ISSUE TITLE
NVDA 2016 fails to read implicit label when type=number

### ISSUE THREAD

**Issue by spanchang** [nvaccess/nvda#5939]
f

**Comment by spanchang** [comment]
Forgot to say this is noted with Firefox 45 on Windows 8.1
This is similar to Bug 5422 filed Oct 2015 for Google Chrome. On testing with Chrome 50, it appears #5422 has been resolved
===
Sailesh.

**Comment by jcsteh** [comment]
Is there still an issue here? If so, the details appear to be missing.

**Comment by RichCaloggero** [comment]
Given this code:

```
<label>My number:
<input type="number"></label>
```

## Expected behavior

- nvda should say "My Number" when control gains focus
- up / down arrow keys should adjust value
- should be able to edit text of number as usual

## Current behavior

Note, as reiterated below, this only seems to happen on desktop machines.  Changing the keyboard setting in NVDA from desktop to laptop has no effect.


- label is not read when control gains focus
	+ label is read if label added via for attribute
- up / down arrows do change value, but in most cases, NVDA loses focus
- text of number can be edited as usual

## Operating system

windows 7, 64-bit

## Browser

Firefox 64-bit latest (54.0.1 at time of writing)


## Hardware

Dell desktop machine

__Note: when using laptop running windows 7, and all other factors equal, NVDA does allow adjusting value via up / down arrows without losing focus. I can think of no reason why this should be!!
However, when input type is "number", NVDA _never_ reads wrapped labels!

**Comment by jcsteh** [comment]
Duplicate of #5917.
